config.yml

```diff
@@ -125,8 +127,6 @@ languages:
 #      accessories:  '/:sections/:slug/'
 #      blog:         '/blog/:url/'
 
-# Preserve special characters in taxonomy names ('Gérard Depardieu' vs 'Gerard Depardieu')
-preserveTaxonomyNames:      false
```

Comment:
what is replacing this? (I tried to find some info but there doesnt seem to be a way to search through Hugo's release notes, at least I did not find any)